ClientConsoleHandler.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.handler;
+
+import mate.academy.dao.Dao;
+import mate.academy.di.Inject;
+import mate.academy.di.Injector;
+import mate.academy.model.Client;
+
+import java.util.Scanner;
+
+public class ClientConsoleHandler implements ConsoleHandler{
+
+    @Inject
+    private static Dao clientDao;
+
+    public static void handle() throws IllegalAccessException {
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("1 - если вносите клиента, 2 - если получаете клиента, 3 - для выхода");

```

Comment:
English please

Reply:
> English please

that's what we wrote in class. :(

Reply:
any russian character in your code decreases chances to get a job or increases chances to lose it